src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs

```diff
@@ -53,12 +53,27 @@ public async Task OnGetAsync(string id)
                 else
                     Comments[comment.ElementId].Add(comment);
             }
+
+            if (User.Identity.IsAuthenticated)
+            {
+                Username = User.FindFirst(c => c.Type == "urn:github:login")?.Value;
+                AvatarUrl = "https://github.com/" + Username + ".png";
```

Comment:
Create a tag helper for rendering avatar that takes a user name, it would clean a lot of things up.
https://docs.microsoft.com/en-us/aspnet/core/mvc/views/tag-helpers/intro?view=aspnetcore-2.2